lib/src/presentation/view/home_page.dart

```diff
@@ -10,20 +20,173 @@ class HomePage extends StatefulWidget {
 }
 
 class _HomePageState extends State<HomePage> {
+  static const String homePageImageRefresh = 'assets/images/refresh.png';
+  static const String homePageImageNoRecipesFound =
+      'assets/images/no_recipes_found.png';
+  static const String homePageTextRefresh = 'SHOW MORE RECIPES';
+  static const String homePageTextNoRecipesFound = 'NO RECIPES FOUND';
+  static const double homePageContainerImageRefresh = 40.0;
+  static const int homePageFirstIndexCard = 0;
+  static const int homePageSecondIndexCard = 1;
+  static const int homePageThirdIndexCard = 2;
+  static const double homePageSizeBox = 30.0;
+  static const double homePageTextNoRecipesFoundStyleSize = 50.0;
+
+  final RecipesController controller = Get.put(
+    RecipesController(
+      RecipesUseCase(
+        RecipesRepository(
+          RecipesApiService(),
+        ),
+      ),
+    ),
+  );
```

Comment:
Instead to define all of this here, you can create a di field, in that file you are going to define, controllers, useCases, apiService, Dio, etc

Reply:
Done